yt_dlp/extractor/bluesky.py

```diff
@@ -0,0 +1,362 @@
+from .common import InfoExtractor
+from ..utils import int_or_none, mimetype2ext, parse_iso8601, traverse_obj, url_or_none
```

Comment:
For my suggestions to work:
```suggestion
from ..utils import (
    format_field,
    int_or_none,
    mimetype2ext,
    parse_iso8601,
    truncate_string,
    update_url_query,
    url_or_none,
    variadic,
)
from ..utils.traversal import traverse_obj
```